app/assets/javascripts/components/common/ArticleViewer/components/IconOpener.jsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export const IconOpener = ({ showArticle, showButtonClass, showButtonLabel }) => (
+export const IconOpener = ({ showArticle, showButtonClass, showButtonLabel, disabled = false }) => (
   <div className={`tooltip-trigger ${showButtonClass}`}>
-    <button onClick={showArticle} aria-label="Open Article Viewer" className="icon icon-article-viewer" />
+    <button 
+      onClick={showArticle}
+      aria-label="Open Article Viewer"
+      className={"icon " + ( disabled ? "icon-article-viewer-disabled" : "icon-article-viewer" ) }
+      disabled={disabled}
+     />
     <div className="tooltip tooltip-center dark large">
-      <p>{showButtonLabel()}</p>
+      {
+        disabled ? <p>Article Doesn't Exist</p> : <p>{showButtonLabel()}</p>
```

Comment:
This message should be done via the i18n system so that it can be translated.

Reply:
ok, i will do it that way